tests/MongoDB.Driver.Tests/IJsonDrivenTestRunner.cs

```diff
@@ -23,6 +23,7 @@ namespace MongoDB.Driver.Tests
 {
     public interface IJsonDrivenTestRunner
     {
+        IMongoClient FailPointClient { get; }
```

Comment:
The fact that this property returns a class from High and the other two methods only use types from Core indicates that this property doesn't belong here.